### PR TITLE
mpd 0.23.9

### PR DESCRIPTION
**Pull request by dvdmuckle**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by carlocab — COMMENTED** [review]
You also need to remove the `__END__` section at the end of the formula.

File: Formula/mpd.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Mpd < Formula
   desc "Music Player Daemon"
   homepage "https://www.musicpd.org/"
-  url "https://www.musicpd.org/download/mpd/0.23/mpd-0.23.8.tar.xz"
-  sha256 "86bb569bf3b519821f36f6bb5564e484e85d2564411b34b200fe2cd3a04e78cf"
+  url "https://www.musicpd.org/download/mpd/0.23/mpd-0.23.9.tar.xz"
+  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```
Comment (carlocab) [inline Formula/mpd.rb:5]: Looks like the checksum changed. I wonder if it was updated in-place.

Comment (dvdmuckle) [reply Formula/mpd.rb]: Replaced, was getting the sha incorrectly on my end.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**